README.rst

```diff
@@ -75,21 +44,25 @@ Activate virtual environment::
 
   source venv/bin/activate
 
+For benchmarking::
+
+  pip install src-python/drbench
+
 Examples
 ========
 
-Build and test examples with gcc on CPU::
+Build and test with gcc for CPU::
 
-  CXX=g++-10 cmake -B build
+  CXX=g++-12 cmake -B build
   make -C build -j all test
 
-Enable SYCL examples::
+Build and test with ipcx for SYCL && CPU::
 
-  CXX=clang++ cmake -B build -DENABLE_SYCL=ON
+  CXX=icpx cmake -B build -DENABLE_SYCL=ON
 
-Enable SYCL-MPI examples::
+Build and test with ipcx for SYCL && CPU on devcloud::
 
-  CXX=clang++ cmake -B build -DENABLE_SYCL_MPI=ON
+  CXX=icpx cmake -B build -DENABLE_SYCL=ON -DENABLE_MPIFORK=on
```

Comment:
If `mpifork` is going to be our default way of running tests (on Intel MPI where as I understand this option is supported) then maybe we should made `MPIFORK=on` as default (if intel MPI is detected) and disabling it should require passing something to cmake. Or not provide any option to cmake, just use fork in tests if intel MPI is found.

I'm afraid about requiring user to pass many options to have our code running in the default way.

Reply:
mpifork is a workaround for a problem on devcloud. It prevents you from doing multi-node so it is not suitable as a default.